injected/scripts/utils/comment-plugin.js

```diff
@@ -0,0 +1,80 @@
+import { promises } from 'node:fs';
+
+/**
+ * @returns {import("esbuild").Plugin}
+ */
+export function commentPlugin() {
+    const PLUGIN_ID = 'comment-override';
+
+    /** @type {import("esbuild").Plugin} */
+    const plugin = {
+        name: PLUGIN_ID,
+        setup(build) {
+            build.onLoad({ filter: /.*/ }, async (args) => {
+                if (!args.path.includes('node_modules')) return undefined;
+                const text = await promises.readFile(args.path, 'utf8');
+                return {
+                    contents: convertToLegalComments(text.toString()),
+                    loader: 'js',
+                };
```

Comment:
this will unfortunately load every every file into memory on the JS side to process it.